### PR TITLE
Fixed: switching on ticketing system doesn't work

### LINKED ISSUE #3676: Switching on "Use Ticketing System" not working
When creating an event the "Switch on Ticketing System" does not display the ticketing option in Chromium.
screenshot from 2017-05-27 15-58-36

### PR DESCRIPTION
**Pull request by poush**
#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream `development` branch.
~The unit tests pass locally with my changes ~
~I have added tests that prove my fix is effective or that my feature works~
~I have added necessary documentation (if appropriate)~

#### Short description of what this resolves:
This happens for the first time when we create event because event tickets array is null and we were rendering template on that null value.

#### Changes proposed in this pull request:

- Create a computed method to check if tickets attribute is not null and returns if tickets to show or not
- Modified template to use that method.
- We were running a forEach loop on null. Added a check there.

Fixes #3676

### CODE REVIEW

File: app/static/js/events/wizard/vue-app.js [hunk 1]
```diff
@@ -387,6 +393,7 @@ function shouldDisableMove($this) {
 
 function checkTickets($this) {
     var flag = false;
+    if($this !== null)
```
Comment (SaptakS) [inline app/static/js/events/wizard/vue-app.js]: Please check indentation here.

File: app/static/js/events/wizard/vue-app.js [hunk 2]
```diff
@@ -387,6 +393,7 @@ function shouldDisableMove($this) {
...
     $this.forEach(function(ticket){
         if(ticket.type === 'paid' && (ticket.price === '' || ticket.price <=0))
         {
```
Comment (SaptakS) [inline app/static/js/events/wizard/vue-app.js]: Please fix this indentation as well. I know maybe it's not your fault. But please refactor this.

File: app/static/js/events/wizard/vue-app.js [hunk 3]
```diff
@@ -61,6 +61,12 @@ var app = new Vue({
             var start = moment(this.event.start_time_date.trim() + " " + this.event.start_time_time.trim(), format);
             var end = moment(this.event.end_time_date.trim() + " " + this.event.end_time_time.trim(), format);
             return end.isBefore(start);
+        },
+        showTickets: function(){
+            if(this.event.tickets === null)
+                return 0;
+            else
+                return this.event.tickets.length > 0
```
Comment (SaptakS) [inline app/static/js/events/wizard/vue-app.js]: Like 79 returns 0 while this line returnss `true` or `false`. Can you try avoiding mixed return types, if possible?

**Comment by poush** [comment]
Thanks, @SaptakS! Updated.

**Pull request merged**